src/main/java/com/minecolonies/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -93,6 +92,19 @@ public EntityAIStructureBuilder(JobBuilder job)
         worker.setCanPickUpLoot(true);
     }
 
+    private boolean checkIfCanceled()
+    {
+        WorkOrderBuild wo = job.getWorkOrder();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'wo' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)